Command/EventLogCleanupCommand.php

```diff
@@ -82,10 +86,19 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             EventLogCleanup::CAMPAIGN_LEAD_EVENTS => $input->getOption('campaign-lead'),
             EventLogCleanup::LEAD_EVENTS          => $input->getOption('lead'),
             EventLogCleanup::EMAIL_STATS          => $input->getOption('email-stats'),
+            EventLogCleanup::EMAIL_STATS_TOKENS   => $input->getOption('email-stats-tokens')
         ];
 
         if (0 === array_sum($operations)) {
             $operations = array_combine(array_keys($operations), array_fill(0, count($operations), true));
+            array_pop($operations);
```

Comment:
Remove this line

Reply:
Thanks @moongazer for the recommendation, that  `array_pop($operations)` is not needed - removed this not harmful but not necessary  part with my current commit.